tests/test_scalar.py

```diff
@@ -286,3 +286,218 @@ def test_validate_scalar_input_correct():
 
     errors = validate(graph, read(query))
     assert errors == []
+
+
+INT_VALUE = 60
+
+
+def test_parse_float_input_correct():
```

Comment:
Maybe these tests are overkill and duplicate checks from `tests/test_validate_query.py`?